.ci/magician/github/membership_data.go

```diff
@@ -2,133 +2,116 @@ package github
 
 import "time"
 
-var (
-	// This is for the random-assignee rotation.
-	reviewerRotation = map[string]struct{}{
-		"slevenick":   {},
-		"c2thorn":     {},
-		"rileykarson": {},
-		"melinath":    {},
-		"ScottSuarez": {},
-		"shuyama1":    {},
-		"roaks3":      {},
-		"zli82016":    {},
-		"trodge":      {},
-		"hao-nan-li":  {},
-		"NickElliot":  {},
-		"BBBmau":      {},
-		"SirGitsalot": {},
+type date struct {
+	year  int
+	month int
+	day   int
+}
+
+func newDate(year, month, day int) date {
+	return date{
+		year:  year,
+		month: month,
+		day:   day,
 	}
+}
 
-	// This is for new team members who are onboarding
-	trustedContributors = map[string]struct{}{}
+type Vacation struct {
+	startDate, endDate date
+}
 
-	// This is for reviewers who are "on vacation": will not receive new review assignments but will still receive re-requests for assigned PRs.
-	// User can specify the time zone like this, and following the example below:
-	pdtLoc, _           = time.LoadLocation("America/Los_Angeles")
-	bstLoc, _           = time.LoadLocation("Europe/London")
-	onVacationReviewers = []onVacationReviewer{
-		// Example: taking vacation from 2024-03-28 to 2024-04-02 in pdt time zone.
-		// both ends are inclusive:
-		// {
-		// 	id:        "xyz",
-		// 	startDate: newDate(2024, 3, 28, pdtLoc),
-		// 	endDate:   newDate(2024, 4, 2, pdtLoc),
-		// },
-		{
-			id:        "BBBmau",
-			startDate: newDate(2024, 11, 1, pdtLoc),
-			endDate:   newDate(2024, 11, 1, pdtLoc),
-		},
-		{
-			id:        "hao-nan-li",
-			startDate: newDate(2024, 12, 23, pdtLoc),
-			endDate:   newDate(2025, 1, 5, pdtLoc),
-		},
-		{
-			id:        "ScottSuarez",
-			startDate: newDate(2024, 4, 30, pdtLoc),
-			endDate:   newDate(2024, 7, 31, pdtLoc),
-		},
-		{
-			id:        "shuyama1",
-			startDate: newDate(2024, 9, 26, pdtLoc),
-			endDate:   newDate(2024, 10, 4, pdtLoc),
-		},
-		{
-			id:        "melinath",
-			startDate: newDate(2024, 12, 19, pdtLoc),
-			endDate:   newDate(2025, 1, 7, pdtLoc),
-		},
-		{
-			id:        "slevenick",
-			startDate: newDate(2024, 7, 5, pdtLoc),
-			endDate:   newDate(2024, 7, 16, pdtLoc),
-		},
-		{
-			id:        "c2thorn",
-			startDate: newDate(2024, 7, 10, pdtLoc),
-			endDate:   newDate(2024, 7, 16, pdtLoc),
+// GetStartDate returns a time corresponding to the beginning of the start date in the given timezone.
+func (v Vacation) GetStartDate(timezone *time.Location) time.Time {
```

Comment:
nit: this file defines a `date` type but `GetStartDate()` and `GetEndDate()` return `time.Time`

Reply:
Removed date from the function names.